project_name/foo.py

```diff
@@ -0,0 +1,3 @@
+def do_something(value):
```

Comment:
We should use the most idiomatic cs school example naming as possible. Bob, Alice, Foo, and Bar.
```suggestion
def bar(value):
```